nest-demo/docker/docker-compose.yml

```
@@ -0,0 +1,129 @@
+    env_file:
+      - .env.jaeger
+  loki:
+    image: grafana/loki:3.5.0
+    container_name: loki
+    restart: unless-stopped
+    env_file:
+      - .env.loki
+    command: -config.file=/etc/loki/loki.yaml
+    user: ":"
+    ports:
+       - "3100:3100"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)